reactNativeProject/src/screens/Main/components/TodoList/layout.js

```diff
@@ -1,33 +1,52 @@
-import React from "react";
+import React, { Component } from "react";
 import { FlatList } from "react-native";
-import { reduxForm } from "redux-form";
+
 import PropTypes from "prop-types";
-import FORM_NAME from "./fields";
+import { connect } from "react-redux";
 import Todo from "./components/Todo";
+import { actionCreators } from "../../../../redux/item/actions";
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
This belongs in the `index.js`. You shouldn´t have deleted it.
When you make the `index.js` again, you just need to `connect` the layout directly instead of connecting the container component. Ask me if you need help with this.